busypie/func.py

```diff
@@ -28,5 +28,7 @@ def _is_a_lambda(func: Callable) -> bool:
 
 def _content_of(lambda_func: Callable) -> str:
     source_line = inspect.getsource(lambda_func)
+    source_line = re.sub(r'\\', '', source_line)
```

Comment:
The problem with this solution is that it will work on strings and comments 
So in order to resolve it we need something more sophisticated. 
On a second thought, I think we can live with it in the description for now, to keep this PR simple 
wdyt?

Reply:
?